backend/core/Extensions/StringExtensions.cs

```diff
@@ -104,5 +104,16 @@ public static string Truncate(this string value, int maxLength)
             if (string.IsNullOrEmpty(value)) return value;
             return value.Length <= maxLength ? value : value.Substring(0, maxLength);
         }
+
+        /// <summary>
+        /// Clean up the project numbers array format and get rid of double quotes, brackets, and commas.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        public static string FormatProjectNumbers(this string value)
+        {
+            if (string.IsNullOrEmpty(value)) return value;
+            return value.Replace("\"", "").Replace("[", "").Replace("]", "").Replace(",", System.Environment.NewLine);
```

Comment:
Replacing "," with a newline probably won't look right when display in HTML.  Not certain what the best solution would be there.  Are you certain you want to format this in the API?  I'd think it would make more sense to format how it looks on the frontend app.

Reply:
I don't think it affects the HTML view, but the newline was for the export to Excel to get the SPP#'s to display on separate lines in Excel.
![image](https://user-images.githubusercontent.com/68400651/139705857-d4ad36a3-6d77-425a-bd51-fc902ac69b49.png)
[pims-inventory (52).xlsx](https://github.com/bcgov/PIMS/files/7454696/pims-inventory.52.xlsx)